### PR TITLE
YLP-2245 Migrate settings print view to dumb

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/dumb/src/modules/print/settings-print-view/settings-print-view.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  timePrefs: TimePrefs
+  deviceMetadata: DeviceMetadata
+
+  constructor(doc: PdfDocumentOverridden, data: PdfSettingsData, params: PrintViewParams) {
+    super(doc, data, params)
+
+    this.source = data.source?.toLowerCase() ?? ''
+
+    const dataTimezone = data.timezone
+    this.timePrefs = dataTimezone ? { timezoneAware: true, timezoneName: dataTimezone } : params.timePrefs
```
[the remark this answers is not in the document]
Comment (celia-rouis) [reply packages/dumb/src/modules/print/settings-print-view/settings-print-view.ts:61]: I don't have this issue but I did update the `TimePrefs` model in my PR. Did you try running `npm run build-dependencies` to get the latest model updates?

File: packages/dumb/src/modules/print/settings-print-view/settings-print-view.util.test.ts [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+      expect(result.heading).toEqual({
+        text: 'Settings'
+      })
+      expect(result.columns).toEqual([{
+        id: 'name',
+        header: 'Name',
+        cache: false,
+        align: 'left',
+        width: 366.29599999999994
```
[the remark this answers is not in the document]
Comment (celia-rouis) [reply packages/dumb/src/modules/print/settings-print-view/settings-print-view.util.test.ts:260]: It's the result of the computation of the column width * 0.7 (see settings-print-view.util.ts line 213)

File: packages/dumb/src/modules/print/settings-print-view/settings-print-view.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (fatb38) [reply packages/dumb/src/modules/print/settings-print-view/settings-print-view.ts:61]: It's ok after `build-dependencies` !! I hadn't thought of doing that 😅

**Review by fatb38 — APPROVED** [review]

**Pull request merged**